### ISSUE TITLE
OpenSSL bump to 1.1.1

### ISSUE THREAD

**Issue by edannenberg** [edannenberg/kubler#168]
Just a quick heads up regarding the recent bump to openssl 1.1.1, you might run into strange net related errors or missing crypto libs. To resolve the issue just delete any related binary packages in `${KUBLER_DATA_DIR}/packages`. (curl and nginx are affected for example)

**Comment by r7l** [comment]
Is there any command that would clean up those packages or any option that would not use any of these binary packages? So far i was always deleting the content of the package directory.

**Comment by r7l** [comment]
I've just updated Kubler again using `kubler update` and i've also deleted all the packages and downloads after running into this issue but it didn't fix it. Every time i am trying to build the the stack from scratch it ends here:

```!!! 'net-misc/openssh' (virtual/ssh) is part of your system profile.
!!! Unmerging it may be damaging to your system.


 net-misc/openssh
    selected: 8.0_p1-r2 
   protected: none 
     omitted: none 

 dev-libs/openssl
    selected: 1.0.2t-r1 
   protected: none 
     omitted: none 

All selected packages: =net-misc/openssh-8.0_p1-r2 =dev-libs/openssl-1.0.2t-r1

>>> 'Selected' packages are slated for removal.
>>> 'Protected' and 'omitted' packages will not be removed.

>>> Waiting 5 seconds before starting...
>>> (Control-C to abort)...
>>> Unmerging in: 5 4 3 2 1
>>> Unmerging (1 of 2) net-misc/openssh-8.0_p1-r2...
>>> Unmerging (2 of 2) dev-libs/openssl-1.0.2t-r1...

!!! existing preserved libs:
>>> package: dev-libs/openssl-1.0.2t-r1
 *  - /usr/lib64/libssl.so.1.0.0
 *      used by /usr/bin/wget (net-misc/wget-1.20.3-r1)
 *      used by /usr/lib64/libcurl.so.4.5.0 (net-misc/curl-7.65.0)
 *      used by /usr/lib64/python2.7/lib-dynload/_ssl.so (dev-lang/python-2.7.15)
 *      used by /usr/lib64/python3.6/lib-dynload/_ssl.cpython-36m-x86_64-linux-gnu.so (dev-lang/python-3.6.5)
 *  - /usr/lib64/libcrypto.so.1.0.0
 *      used by /bin/ping (net-misc/iputils-20180629)
 *      used by /usr/bin/q (app-portage/portage-utils-0.80)
 *      used by /usr/bin/wget (net-misc/wget-1.20.3-r1)
 *      used by 5 other files
Use emerge @preserved-rebuild to rebuild packages using these libraries
At argv[2]='+~amd64': ~amd64 seems to be an incorrect keyword for =dev-libs/openssl-1.1.1c-r1
Strict mode, aborting.
»[✘]»[kubler/bob]» fatal: Failed to run image kubler/bob-core:20190420
»[✘]»[kubler/bob]» removing rootfs-builder-kubler-bob-9295-2763, NO_CLEANUP env prevents this```

**Comment by edannenberg** [comment]
```
At argv[2]='+~amd64': ~amd64 seems to be an incorrect keyword for =dev-libs/openssl-1.1.1c-r1
```

Not related, this is a flaggie (the tool used by `update_use/keywords`) issue, much like the `sed-or-die` helper it will fail if a referenced version is no longer in the portage tree or moved to stable.

Bad timing as I just pushed the monthly update a couple hours before the upstream bump. Thanks for reporting, fix is up.

**Comment by r7l** [comment]
Ya i knew it wasn't exactly the same issue. Should have added new issue. Thanks for quick fixing!
